src/main.ts

```diff
@@ -56,20 +57,19 @@ export async function run() {
       return Promise.reject(new Error('Not Supported Flow'))
     }
 
-    if (inputs.POLARIS_SERVER_URL) {
+    if (inputs.POLARIS_SERVER_URL && validatePolarisInputs()) {
```

Comment:
inputs.POLARIS_SERVER_URL  can't we check that inside ValidatePolarisInputs . same for others?